packages/c/cpuinfo/xmake.lua

```diff
@@ -11,7 +11,6 @@ package("cpuinfo")
     on_install(function (package)
         local configs = {}
         table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
-        table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
```

Comment:
why?

Reply:
we need it.

Reply:
if it do not shared or static, you need set

https://github.com/xmake-io/xmake-repo/blob/0fd68bc72a2172fe15f5939e90b7cc4fe439cd35/packages/v/vc/xmake.lua#L10

Reply:
It failed on the CI and on my computer

Reply:
you should fix it or add `add_configs` to disable it.

Reply:
and it only for windows, but you disable them for all platforms.